  INFO root [2018-10-31 22:24:49,386] - 
  INFO root [2018-10-31 22:24:49,406] - -----------------------------------------------------------------------------------------


  INFO root [2018-10-31 22:24:50,882] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [2018-10-31 22:24:50,883] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [2018-10-31 22:24:50,884] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [2018-10-31 22:24:51,396] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.107 ( 192.168.0.107:65288 )
  INFO root [2018-10-31 22:24:51,398] - NetworkInterface::initialize(INTERNAL): address 192.168.0.107:65288, SOMAXCONN=5.
  INFO root [2018-10-31 22:24:51,398] - server-configs:
  INFO root [2018-10-31 22:24:51,399] - 	internalAddr : 192.168.0.107:65288
  INFO root [2018-10-31 22:24:51,399] - 	componentID : 4000
  INFO root [2018-10-31 22:24:53,143] - ---- dbmgr Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: F28F1C37BCC70CD27B77841D4A30E3D7. Config: Debug X86. Built: 15:52:44 Apr 29 2018. AppID: 4000. UID: 29613. PID: 5772 ----
  INFO root [2018-10-31 22:24:53,145] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [2018-10-31 22:24:53,168] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-10-31 22:24:53,186] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-10-31 22:24:57,114] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-10-31 22:24:57,332] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-10-31 22:24:57,403] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-10-31 22:24:58,281] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-10-31 22:24:58,299] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-10-31 22:24:58,301] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-10-31 22:24:58,460] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-10-31 22:24:59,526] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-10-31 22:25:00,198] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:52:37) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-10-31 22:25:00,199] - Script::install(): KBExtra is successfully!
  INFO root [2018-10-31 22:25:01,789] - Dbmgr::initializeEnd: digest(7F4FC8296AD553545CD95B23EFF6238F)
S_INFO root [2018-10-31 22:25:01,790] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-10-31 22:25:01,848] - TelnetServer server is running on port 32000
  INFO root [2018-10-31 22:25:01,875] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2018-10-31 22:25:02,377] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
AAE96C3A380726D06B152EBC1FBA5C6C259350C1F6277DB987F274C61760E2712E405BBD306CCDA425DD56B3D264F681F904D4C7911D348449C30858C30F68C20459C2BE90FA2A44E01A7AEDF4A14FCBF76219CF4A2E61366E7B8D02A558822CF977CC241C6AB9138CBADE889E814E45FD1061AC1E911B73303A91C65E452457
 DEBUG root [2018-10-31 22:25:04,150] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2018-10-31 22:25:06,112] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2018-10-31 22:25:06,114] - DBUtil::createInterface[0x34f3d4]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2018-10-31 22:25:06,414] - ThreadPool::createThreadPool: creating  threadpool...
 DEBUG root [2018-10-31 22:25:06,415] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2018-10-31 22:25:06,415] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2018-10-31 22:25:06,416] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
  INFO root [2018-10-31 22:25:06,416] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2018-10-31 22:25:06,417] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2018-10-31 22:25:06,418] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2018-10-31 22:25:06,421] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2018-10-31 22:25:06,422] - DBThread::onStart(): 0x49de70!
 DEBUG root [2018-10-31 22:25:06,423] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2018-10-31 22:25:06,423] - DBThread::onStart(): 0x49df80!
 DEBUG root [2018-10-31 22:25:06,423] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2018-10-31 22:25:06,424] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2018-10-31 22:25:06,424] - DBThread::onStart(): 0x49def8!
 DEBUG root [2018-10-31 22:25:06,425] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2018-10-31 22:25:06,425] - DBThread::onStart(): 0x49e090!
 DEBUG root [2018-10-31 22:25:06,426] - DBThread::onStart(): 0x49e008!
  INFO root [2018-10-31 22:25:06,801] - DBInterfaceMysql::query: successfully!
  INFO root [2018-10-31 22:25:06,803] - DBInterfaceMysql::query: successfully!
  INFO root [2018-10-31 22:25:07,351] - DBInterfaceMysql::query: successfully!
  INFO root [2018-10-31 22:25:07,520] - DBInterfaceMysql::query: successfully!
  WARN root [2018-10-31 22:25:07,659] - DBTask::process(): took 0.54 seconds
sql:(show index from tbl_CheckName)
  INFO root [2018-10-31 22:25:07,751] - DBInterfaceMysql::query: successfully!
  INFO root [2018-10-31 22:25:07,752] - DBInterfaceMysql::query: successfully!
  WARN root [2018-10-31 22:25:07,753] - DBTask::process(): took 0.86 seconds
sql:(CREATE TABLE IF NOT EXISTS kbe_accountinfos (`accountName` varchar(128) not null, PRIMARY KEY idKey (`accountName`),`password` varchar(255),`bindata` blob,`email` varchar(191) not null, UNIQUE KEY `email` (`email`),`entityDBID` bigint(20) unsigned not null DEFAULT 0, UNIQUE KEY `entityDBID` (`entityDBID`),`flags` int unsigned not null DEFAULT 0,`deadline` bigint(20) not null DEFAULT 0,`regtime` bigint(20) not null DEFAULT 0,`lasttime` bigint(20) not null DEFAULT 0,`numlogin` int unsigned not null DEFAULT 0)ENGINE=InnoDB)
  WARN root [2018-10-31 22:25:08,005] - DBTask::process(): took 1.11 seconds
sql:(insert into kbe_serverlog (heartbeatTime, logger) values(1540995907,4000) ON DUPLICATE KEY UPDATE heartbeatTime=1540995907)
  INFO root [2018-10-31 22:25:08,127] - DBInterfaceMysql::query: successfully!
  WARN root [2018-10-31 22:25:08,128] - DBTask::process(): took 1.23 seconds
sql:(delete from kbe_entitylog where logger in (4000))
  INFO root [2018-10-31 22:25:08,310] - DBInterfaceMysql::query: successfully!
  INFO root [2018-10-31 22:25:08,311] - DBInterfaceMysql::query: successfully!
  INFO root [2018-10-31 22:25:08,312] - DBInterfaceMysql::query: successfully!
  WARN root [2018-10-31 22:25:08,313] - DBTask::process(): took 1.42 seconds
sql:(delete from kbe_email_verification where logtime<1540992308 and type=3)
  WARN root [2018-10-31 22:25:08,314] - DBTask::process(): took 0.65 seconds
sql:(show index from tbl_Account)
  WARN root [2018-10-31 22:25:08,516] - DBTask::process(): took 0.76 seconds
sql:(show index from tbl_Halls)
  WARN root [2018-10-31 22:25:08,671] - DBTask::process(): delay 1.20 seconds, try adjusting the kbengine[_defs].xml(numConnections) and MySQL(my.cnf->max_connections or innodb_flush_log_at_trx_commit)!
sql:(show index from tbl_Room)
  WARN root [2018-10-31 22:25:08,672] - DBTask::process(): took 0.67 seconds
sql:(show index from tbl_Room)
  WARN root [2018-10-31 22:25:08,694] - e:\kbengine\kbe\src\server\dbmgr\dbmgr.cpp:275: Profile scriptCall took 6.90 seconds
  INFO root [2018-10-31 22:25:09,223] - ---- dbmgr is running ----
 DEBUG root [2018-10-31 22:25:09,390] - Components::process(): Request for the process of identity...
  INFO root [2018-10-31 22:25:16,009] - ServerApp::onRegisterNewApp: uid:29613, username:, componentType:baseappmgr, componentID:5000, globalorderID=-1, grouporderID=-1, intaddr:192.168.0.107, intport:65290, extaddr:nonsupport, extport:0,  from 192.168.0.107:65309/0/0/0.
  INFO root [2018-10-31 22:25:16,010] - Components::addComponent: reset orderLog, uid=29613!
  INFO root [2018-10-31 22:25:16,010] - Components::addComponent[baseappmgr], uid=29613, componentID=5000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [2018-10-31 22:25:16,011] - ServerApp::onRegisterNewApp: uid:29613, username:, componentType:cellappmgr, componentID:6000, globalorderID=-1, grouporderID=-1, intaddr:192.168.0.107, intport:65287, extaddr:nonsupport, extport:0,  from 192.168.0.107:65311/0/0/0.
  INFO root [2018-10-31 22:25:16,011] - Components::addComponent[cellappmgr], uid=29613, componentID=6000, globalorderid=2, grouporderid=1, totalcount=1
  INFO root [2018-10-31 22:25:17,009] - Components::findComponents: find logger(1)...
  INFO root [2018-10-31 22:25:17,024] - Components::findComponents: found logger, addr:192.168.0.107:65285
  INFO root [2018-10-31 22:25:17,025] - Components::addComponent[logger], uid=29613, componentID=2000, globalorderid=3, grouporderid=1, totalcount=1
  WARN root [2018-10-31 22:25:17,108] - Forwarding logs to logger[192.168.0.107:65285/0/0/0]...
 ERROR root [2018-10-31 22:30:03,231] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2018-10-31 22:30:03,234] - The following logs sent to logger failed:
  WARN root [2018-10-31 22:30:03,235] - ==> [2018-10-31 22:30:03 230] TCPPacketReceiver::processPendingEvents(192.168.0.107:65285): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-10-31 22:30:03,235] - ==> [2018-10-31 22:30:03 231] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2018-10-31 22:30:04,547] - 
  INFO root [2018-10-31 22:30:04,548] - -----------------------------------------------------------------------------------------


  INFO root [2018-10-31 22:30:04,549] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [2018-10-31 22:30:04,549] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [2018-10-31 22:30:04,549] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [2018-10-31 22:30:04,553] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.107 ( 192.168.0.107:65429 )
  INFO root [2018-10-31 22:30:04,553] - NetworkInterface::initialize(INTERNAL): address 192.168.0.107:65429, SOMAXCONN=5.
  INFO root [2018-10-31 22:30:04,553] - server-configs:
  INFO root [2018-10-31 22:30:04,553] - 	internalAddr : 192.168.0.107:65429
  INFO root [2018-10-31 22:30:04,553] - 	componentID : 4000
  WARN root [2018-10-31 22:30:04,869] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20592, errorno=10048
  INFO root [2018-10-31 22:30:04,872] - ---- dbmgr Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: F28F1C37BCC70CD27B77841D4A30E3D7. Config: Debug X86. Built: 15:52:44 Apr 29 2018. AppID: 4000. UID: 30694. PID: 696 ----
  INFO root [2018-10-31 22:30:04,872] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [2018-10-31 22:30:04,873] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-10-31 22:30:04,873] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-10-31 22:30:05,000] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-10-31 22:30:05,001] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-10-31 22:30:05,001] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-10-31 22:30:05,004] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-10-31 22:30:05,004] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-10-31 22:30:05,004] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-10-31 22:30:05,005] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-10-31 22:30:05,035] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-10-31 22:30:05,066] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:52:37) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-10-31 22:30:05,066] - Script::install(): KBExtra is successfully!
  INFO root [2018-10-31 22:30:05,095] - Dbmgr::initializeEnd: digest(7F4FC8296AD553545CD95B23EFF6238F)
S_INFO root [2018-10-31 22:30:05,096] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-10-31 22:30:05,096] - TelnetServer server is running on port 32000
  INFO root [2018-10-31 22:30:05,096] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2018-10-31 22:30:05,429] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
00949837FFBBC7B4117C434D40DABE75F0572FA85CDBFB80E1EA50D23D7935A0B9EB55738B7ED7D7FA737CFE28705C9C02BC181D7959D936B2B26A6FD59F1DD094BDA8B29BF95A33808AC6FA0F5E3A0839B8E1B6BC4BD44FC235F32A314906DD80B3341075F61AEA480A856C8BA2BF30C20A34C59B571A102850C8660F60AE29
 DEBUG root [2018-10-31 22:30:05,430] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2018-10-31 22:30:05,432] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2018-10-31 22:30:05,433] - DBUtil::createInterface[0x181ee4c]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2018-10-31 22:30:05,433] - ThreadPool::createThreadPool: creating  threadpool...
 DEBUG root [2018-10-31 22:30:05,433] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2018-10-31 22:30:05,433] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2018-10-31 22:30:05,433] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2018-10-31 22:30:05,433] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
  INFO root [2018-10-31 22:30:05,434] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2018-10-31 22:30:05,434] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2018-10-31 22:30:05,435] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2018-10-31 22:30:05,435] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2018-10-31 22:30:05,435] - DBThread::onStart(): 0x1b3dde8!
 DEBUG root [2018-10-31 22:30:05,435] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2018-10-31 22:30:05,435] - DBThread::onStart(): 0x1b3dd60!
 DEBUG root [2018-10-31 22:30:05,435] - DBThread::onStart(): 0x1b3de70!
 DEBUG root [2018-10-31 22:30:05,435] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2018-10-31 22:30:05,435] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2018-10-31 22:30:05,436] - DBThread::onStart(): 0x1b3df80!
 DEBUG root [2018-10-31 22:30:05,436] - DBThread::onStart(): 0x1b3def8!
  INFO root [2018-10-31 22:30:05,535] - DBInterfaceMysql::query: successfully!
  INFO root [2018-10-31 22:30:05,536] - DBInterfaceMysql::query: successfully!
  INFO root [2018-10-31 22:30:05,537] - DBInterfaceMysql::query: successfully!
  INFO root [2018-10-31 22:30:05,537] - DBInterfaceMysql::query: successfully!
  INFO root [2018-10-31 22:30:05,537] - DBInterfaceMysql::query: successfully!
  INFO root [2018-10-31 22:30:05,537] - DBInterfaceMysql::query: successfully!
  INFO root [2018-10-31 22:30:05,537] - DBInterfaceMysql::query: successfully!
  INFO root [2018-10-31 22:30:05,537] - DBInterfaceMysql::query: successfully!
  INFO root [2018-10-31 22:30:05,538] - DBInterfaceMysql::query: successfully!
  INFO root [2018-10-31 22:30:05,538] - DBInterfaceMysql::query: successfully!
  WARN root [2018-10-31 22:30:05,576] - e:\kbengine\kbe\src\server\dbmgr\dbmgr.cpp:275: Profile scriptCall took 0.48 seconds
  INFO root [2018-10-31 22:30:05,860] - ---- dbmgr is running ----
 DEBUG root [2018-10-31 22:30:05,860] - Components::process(): Request for the process of identity...
  INFO root [2018-10-31 22:30:12,976] - Components::findComponents: find logger(1)...
  INFO root [2018-10-31 22:30:13,064] - Components::findComponents: found logger, addr:192.168.0.107:65427
  INFO root [2018-10-31 22:30:13,066] - Components::addComponent: reset orderLog, uid=30694!
  INFO root [2018-10-31 22:30:13,066] - Components::addComponent[logger], uid=30694, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  WARN root [2018-10-31 22:30:13,068] - Forwarding logs to logger[192.168.0.107:65427/0/0/0]...
 ERROR root [2018-10-31 22:33:32,730] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2018-10-31 22:33:32,735] - The following logs sent to logger failed:
  WARN root [2018-10-31 22:33:32,735] - ==> [2018-10-31 22:33:32 727] TCPPacketReceiver::processPendingEvents(192.168.0.107:65427): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-10-31 22:33:32,735] - ==> [2018-10-31 22:33:32 730] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2018-10-31 22:33:34,001] - 
  INFO root [2018-10-31 22:33:34,003] - -----------------------------------------------------------------------------------------


  INFO root [2018-10-31 22:33:34,004] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [2018-10-31 22:33:34,004] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [2018-10-31 22:33:34,004] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [2018-10-31 22:33:34,009] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.107 ( 192.168.0.107:49171 )
  INFO root [2018-10-31 22:33:34,009] - NetworkInterface::initialize(INTERNAL): address 192.168.0.107:49171, SOMAXCONN=5.
  INFO root [2018-10-31 22:33:34,009] - server-configs:
  INFO root [2018-10-31 22:33:34,009] - 	internalAddr : 192.168.0.107:49171
  INFO root [2018-10-31 22:33:34,009] - 	componentID : 4000
  INFO root [2018-10-31 22:33:35,526] - ---- dbmgr Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: F28F1C37BCC70CD27B77841D4A30E3D7. Config: Debug X86. Built: 15:52:44 Apr 29 2018. AppID: 4000. UID: 31377. PID: 9632 ----
  INFO root [2018-10-31 22:33:35,528] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [2018-10-31 22:33:35,528] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-10-31 22:33:35,529] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-10-31 22:33:35,685] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-10-31 22:33:35,686] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-10-31 22:33:35,686] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-10-31 22:33:35,689] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-10-31 22:33:35,690] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-10-31 22:33:35,690] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-10-31 22:33:35,690] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-10-31 22:33:35,718] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-10-31 22:33:35,745] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:52:37) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-10-31 22:33:35,745] - Script::install(): KBExtra is successfully!
  INFO root [2018-10-31 22:33:35,763] - Dbmgr::initializeEnd: digest(7F4FC8296AD553545CD95B23EFF6238F)
S_INFO root [2018-10-31 22:33:35,763] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-10-31 22:33:35,764] - TelnetServer server is running on port 32000
  INFO root [2018-10-31 22:33:35,764] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2018-10-31 22:33:36,099] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
29279F528B6B0ACDCB33EF1458E4B5174507B5183E191AFA534761D6B8B18232A2E2A31271FE445EC325FC674DAAB459ACBB876BAAACCD5F449862A4B0E634447CF67B0DF79B1581DA50B26CE98032B9BDAD14085AD8BEDB8B2E354DCFA019F65315DEE8FD9ABE168FDC85A5A5F8910344E87978890CDB7BEEAAAB43C39EB207
 DEBUG root [2018-10-31 22:33:36,101] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2018-10-31 22:33:36,103] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2018-10-31 22:33:36,103] - DBUtil::createInterface[0x184ef40]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2018-10-31 22:33:36,104] - ThreadPool::createThreadPool: creating  threadpool...
 DEBUG root [2018-10-31 22:33:36,104] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2018-10-31 22:33:36,104] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2018-10-31 22:33:36,104] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
  INFO root [2018-10-31 22:33:36,104] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2018-10-31 22:33:36,105] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2018-10-31 22:33:36,105] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2018-10-31 22:33:36,105] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2018-10-31 22:33:36,106] - DBThread::onStart(): 0x1b8de70!
 DEBUG root [2018-10-31 22:33:36,106] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2018-10-31 22:33:36,106] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2018-10-31 22:33:36,106] - DBThread::onStart(): 0x1b8def8!
 DEBUG root [2018-10-31 22:33:36,106] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2018-10-31 22:33:36,106] - DBThread::onStart(): 0x1b8df80!
 DEBUG root [2018-10-31 22:33:36,107] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2018-10-31 22:33:36,107] - DBThread::onStart(): 0x1b8e090!
 DEBUG root [2018-10-31 22:33:36,107] - DBThread::onStart(): 0x1b8e008!
  INFO root [2018-10-31 22:33:36,206] - DBInterfaceMysql::query: successfully!
  INFO root [2018-10-31 22:33:36,207] - DBInterfaceMysql::query: successfully!
  INFO root [2018-10-31 22:33:36,208] - DBInterfaceMysql::query: successfully!
  INFO root [2018-10-31 22:33:36,208] - DBInterfaceMysql::query: successfully!
  INFO root [2018-10-31 22:33:36,208] - DBInterfaceMysql::query: successfully!
  INFO root [2018-10-31 22:33:36,208] - DBInterfaceMysql::query: successfully!
  INFO root [2018-10-31 22:33:36,209] - DBInterfaceMysql::query: successfully!
  INFO root [2018-10-31 22:33:36,209] - DBInterfaceMysql::query: successfully!
  INFO root [2018-10-31 22:33:36,209] - DBInterfaceMysql::query: successfully!
  INFO root [2018-10-31 22:33:36,209] - DBInterfaceMysql::query: successfully!
  WARN root [2018-10-31 22:33:36,257] - e:\kbengine\kbe\src\server\dbmgr\dbmgr.cpp:275: Profile scriptCall took 0.49 seconds
  INFO root [2018-10-31 22:33:36,560] - ---- dbmgr is running ----
 DEBUG root [2018-10-31 22:33:36,560] - Components::process(): Request for the process of identity...
  INFO root [2018-10-31 22:33:43,683] - Components::findComponents: find logger(1)...
  INFO root [2018-10-31 22:33:43,744] - Components::findComponents: found logger, addr:192.168.0.107:49178
  INFO root [2018-10-31 22:33:43,745] - Components::addComponent: reset orderLog, uid=31377!
  INFO root [2018-10-31 22:33:43,746] - Components::addComponent[logger], uid=31377, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  WARN root [2018-10-31 22:33:43,748] - Forwarding logs to logger[192.168.0.107:49178/0/0/0]...
 ERROR root [2018-10-31 22:58:35,850] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2018-10-31 22:58:35,855] - The following logs sent to logger failed:
  WARN root [2018-10-31 22:58:35,855] - ==> [2018-10-31 22:58:35 848] TCPPacketReceiver::processPendingEvents(192.168.0.107:49178): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-10-31 22:58:35,855] - ==> [2018-10-31 22:58:35 850] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2018-10-31 23:03:36,303] - 
  INFO root [2018-10-31 23:03:36,304] - -----------------------------------------------------------------------------------------


  INFO root [2018-10-31 23:03:36,305] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [2018-10-31 23:03:36,305] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [2018-10-31 23:03:36,305] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [2018-10-31 23:03:36,317] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.107 ( 192.168.0.107:50005 )
  INFO root [2018-10-31 23:03:36,317] - NetworkInterface::initialize(INTERNAL): address 192.168.0.107:50005, SOMAXCONN=5.
  INFO root [2018-10-31 23:03:36,317] - server-configs:
  INFO root [2018-10-31 23:03:36,317] - 	internalAddr : 192.168.0.107:50005
  INFO root [2018-10-31 23:03:36,317] - 	componentID : 4000
  INFO root [2018-10-31 23:03:37,829] - ---- dbmgr Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: F28F1C37BCC70CD27B77841D4A30E3D7. Config: Debug X86. Built: 15:52:44 Apr 29 2018. AppID: 4000. UID: 4493. PID: 11476 ----
  INFO root [2018-10-31 23:03:37,830] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [2018-10-31 23:03:37,830] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [2018-10-31 23:03:37,831] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [2018-10-31 23:03:37,973] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [2018-10-31 23:03:37,973] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [2018-10-31 23:03:37,973] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [2018-10-31 23:03:37,977] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [2018-10-31 23:03:37,977] - Script::install_py_dlls(): PyInit_select
 DEBUG root [2018-10-31 23:03:37,978] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [2018-10-31 23:03:37,978] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [2018-10-31 23:03:38,008] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [2018-10-31 23:03:38,036] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:52:37) [MSC v.1913 32 bit (Intel)])!
  INFO root [2018-10-31 23:03:38,037] - Script::install(): KBExtra is successfully!
  INFO root [2018-10-31 23:03:38,055] - Dbmgr::initializeEnd: digest(7F4FC8296AD553545CD95B23EFF6238F)
S_INFO root [2018-10-31 23:03:38,055] - onDBMgrReady: bootstrapGroupIndex=0, bootstrapGlobalIndex=0
  INFO root [2018-10-31 23:03:38,056] - TelnetServer server is running on port 32000
  INFO root [2018-10-31 23:03:38,056] - Dbmgr::initInterfacesHandler: interfaces addr(127.0.0.1:30099), accountType:(interfaces), chargeType:(interfaces).
  WARN root [2018-10-31 23:03:38,411] - DBUtil::initialize: db(default) password is not encrypted!
please use password(rsa):
6EB1C6EEF60CE735FA43A315C6AB6A979C347FAC65F0727BF896612854607FFE5F9A447899A834DD3C3AAF92E7D9F090101C2AAF973041A408816140B7B584E885695F356EF52A21835B7EC8E8AE08AD92F9D48BB764D44E89AB8F11C3901803CCBF0E528C2E902D1412B59DBAE54BFA0EE2348E72A4CD0472897EAD3D35FE2D
 DEBUG root [2018-10-31 23:03:38,412] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2018-10-31 23:03:38,414] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
  INFO root [2018-10-31 23:03:38,414] - DBUtil::createInterface[0x24f230]: interface=default, dbtype=mysql, ip=localhost, port=3306, currdatabase=kbe, username=kbe, connected=yes.

  INFO root [2018-10-31 23:03:38,414] - ThreadPool::createThreadPool: creating  threadpool...
 DEBUG root [2018-10-31 23:03:38,415] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2018-10-31 23:03:38,415] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2018-10-31 23:03:38,415] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
  INFO root [2018-10-31 23:03:38,415] - ThreadPool::createThreadPool: successfully(5), newThreadCount=5, normalMaxThreadCount=5, maxThreadCount=5
 DEBUG root [2018-10-31 23:03:38,415] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2018-10-31 23:03:38,415] - DBInterfaceMysql::attach: connect: localhost:3306 starting...
 DEBUG root [2018-10-31 23:03:38,416] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2018-10-31 23:03:38,416] - DBThread::onStart(): 0x5f8f48!
 DEBUG root [2018-10-31 23:03:38,417] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2018-10-31 23:03:38,417] - DBThread::onStart(): 0x5f8fd0!
 DEBUG root [2018-10-31 23:03:38,417] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2018-10-31 23:03:38,417] - DBThread::onStart(): 0x5f9058!
 DEBUG root [2018-10-31 23:03:38,417] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2018-10-31 23:03:38,417] - DBInterfaceMysql::attach: successfully! addr: localhost:3306
 DEBUG root [2018-10-31 23:03:38,417] - DBThread::onStart(): 0x5f90e0!
 DEBUG root [2018-10-31 23:03:38,417] - DBThread::onStart(): 0x5f9168!
  INFO root [2018-10-31 23:03:38,516] - DBInterfaceMysql::query: successfully!
  INFO root [2018-10-31 23:03:38,517] - DBInterfaceMysql::query: successfully!
  INFO root [2018-10-31 23:03:38,518] - DBInterfaceMysql::query: successfully!
  INFO root [2018-10-31 23:03:38,518] - DBInterfaceMysql::query: successfully!
  INFO root [2018-10-31 23:03:38,518] - DBInterfaceMysql::query: successfully!
  INFO root [2018-10-31 23:03:38,518] - DBInterfaceMysql::query: successfully!
  INFO root [2018-10-31 23:03:38,519] - DBInterfaceMysql::query: successfully!
  INFO root [2018-10-31 23:03:38,519] - DBInterfaceMysql::query: successfully!
  INFO root [2018-10-31 23:03:38,519] - DBInterfaceMysql::query: successfully!
  INFO root [2018-10-31 23:03:38,574] - DBInterfaceMysql::query: successfully!
  WARN root [2018-10-31 23:03:38,618] - e:\kbengine\kbe\src\server\dbmgr\dbmgr.cpp:275: Profile scriptCall took 0.56 seconds
  INFO root [2018-10-31 23:03:38,922] - ---- dbmgr is running ----
 DEBUG root [2018-10-31 23:03:38,922] - Components::process(): Request for the process of identity...
  INFO root [2018-10-31 23:03:46,035] - Components::findComponents: find logger(1)...
  INFO root [2018-10-31 23:03:46,128] - Components::findComponents: found logger, addr:192.168.0.107:50004
  INFO root [2018-10-31 23:03:46,128] - Components::addComponent: reset orderLog, uid=4493!
  INFO root [2018-10-31 23:03:46,128] - Components::addComponent[logger], uid=4493, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  WARN root [2018-10-31 23:03:46,129] - Forwarding logs to logger[192.168.0.107:50004/0/0/0]...
 ERROR root [2018-10-31 23:09:45,794] - Components::removeComponentByChannel: cellapp : 8001, Abnormal exit! errorno=10054
 ERROR root [2018-10-31 23:09:45,803] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [2018-10-31 23:09:45,809] - The following logs sent to logger failed:
  WARN root [2018-10-31 23:09:45,809] - ==> [2018-10-31 23:09:45 787] TCPPacketReceiver::processPendingEvents(192.168.0.107:50036): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-10-31 23:09:45,809] - ==> [2018-10-31 23:09:45 796] Components::removeComponentByChannel: cellapp : 8001, Abnormal exit! errorno=10054
  WARN root [2018-10-31 23:09:45,810] - ==> [2018-10-31 23:09:45 799] TCPPacketReceiver::processPendingEvents(192.168.0.107:50004): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-10-31 23:09:45,810] - ==> [2018-10-31 23:09:45 804] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [2018-10-31 23:09:45,810] - TCPPacketReceiver::processPendingEvents(192.168.0.107:50025): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-10-31 23:09:45,812] - Components::removeComponentByChannel: baseappmgr : 5000, Abnormal exit! errorno=10054
  WARN root [2018-10-31 23:09:45,813] - TCPPacketReceiver::processPendingEvents(192.168.0.107:50028): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-10-31 23:09:45,814] - Components::removeComponentByChannel: cellappmgr : 6000, Abnormal exit! errorno=10054
  WARN root [2018-10-31 23:09:45,822] - TCPPacketReceiver::processPendingEvents(192.168.0.107:50031): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-10-31 23:09:45,823] - Components::removeComponentByChannel: loginapp : 9000, Abnormal exit! errorno=10054
  WARN root [2018-10-31 23:09:45,824] - TCPPacketReceiver::processPendingEvents(192.168.0.107:50033): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [2018-10-31 23:09:45,825] - Components::removeComponentByChannel: baseapp : 7001, Abnormal exit! errorno=10054
